src/v/features/enterprise_features.cc

```diff
@@ -37,6 +37,8 @@ std::ostream& operator<<(std::ostream& os, license_required_feature f) {
         return os << "rbac";
     case license_required_feature::fips:
         return os << "fips";
+    case license_required_feature::datalake:
+        return os << "datalake";
```

Comment:
Is this visible to the customer? Should we call this iceberg??

Reply:
I agree with Tyler, let's try to keep naming consistent

Reply:
I'm not sure if this is user visible, renamed to "datalake_iceberg".. rationale being datalake is the overarching feature while iceberg is an implementation detail.

Reply:
I believe it's user visible. I'm not sure our product messaging has made it clear that iceberg is an implementation detail :)

Reply:
lol I think the code is fine as is :)